### ISSUE TITLE
Installation error during "configure/autoconf" step

### ISSUE THREAD

**Issue by LittleZen** [bnoordhuis/tkgate#8]
Hi, 

I was looking to build tkgate 2.0, but I found some errors during the installation phase! Unfortunately I wasn't able to install the program. I've followed all the steps wrote in the "INSTALL" file, and I've also correctly installed all the necessary dependencies. I've also tried to manually setup the ./config file in order to correctly read the *.sh files, but still got errors. That's the screenshots about the problem:

![image](https://user-images.githubusercontent.com/35767830/153902605-488b7faa-e97c-449d-afe3-70e7b78b1ddf.png)

Basically the terminal return the error `configure: error: could not find tclConfig.sh` during the launch of the `./config` script!

**Comment by bnoordhuis** [comment]
tkgate's configure script doesn't use `--with-tclConfig-file=...` and `--with-tkConfig-file=...`, as far as I know, but you can point it to your tcl/tk install with the `TKGATE_INCDIRS` and `TKGATE_LIBDIRS` environment variables: https://github.com/bnoordhuis/tkgate/blob/5dbabc25c03da19b0872ec399fac9ed9c2ea7ede/Makefile.macos#L19
`$(DESTDIR)/include` and `$(DESTDIR)/lib` contain the tcl/tk header files and libraries respectively in this example.
